src/styles/pages/admin/_material-requests.scss

```diff
@@ -0,0 +1,23 @@
+@use "src/styles/abstracts" as *;
+
+$page: "p-admin-material-requests";
+
+.#{$page} {
```

Comment:
Shift layout on mobile devices. I suggest to switch to `flex-direction: column`. Maybe take a look at how it's done for the CP material requests so you can put the elements in the same order ;)